### PR TITLE
Add reference to rename endpoint in correct version & add to new

### PR DESCRIPTION
**Pull request by jessfraz**
Docker-DCO-1.1-Signed-off-by: Jessica Frazelle jess@docker.com (github: jfrazelle)

### CODE REVIEW

**Comment by jfrazelle** [comment]
ping @vieux :D

File: docs/sources/reference/api/docker_remote_api.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ Docker client now hints potential proxies about connection hijacking using HTTP
 **New!**
 This endpoint now returns the list current execs associated with the container (`ExecIDs`).
 
+`POST /containers/(id)/rename/(new_name)`
```
Comment (vieux) [inline docs/sources/reference/api/docker_remote_api.md]: it's `POST /containers/(id)/rename`

File: docs/sources/reference/api/docker_remote_api_v1.17.md [hunk 2]
```diff
@@ -599,6 +599,27 @@ Status Codes:
...
 
+### Rename a container
+
+`POST /containers/(id)/rename/(new_name)`
+
+Rename the container `id` to a `new_name`
+
+**Example request**:
+
+        POST /containers/e90e34656806/rename/new_name HTTP/1.1
```
Comment (vieux) [inline docs/sources/reference/api/docker_remote_api_v1.17.md]: it's `POST /containers/e90e34656806/rename?name=new_name HTTP/1.1`

File: docs/sources/reference/api/docker_remote_api_v1.17.md [hunk 3]
```diff
@@ -599,6 +599,27 @@ Status Codes:
...
+
+**Example response**:
+
+        HTTP/1.1 204 No Content
+
+Status Codes:
```
Comment (vieux) [inline docs/sources/reference/api/docker_remote_api_v1.17.md]: It's missing the  `Query Parameters` section

**Comment by jfrazelle** [comment]
whoa the other docs were wrong then, updating

**Comment by jfrazelle** [comment]
good catch thanks, updated

**Comment by vieux** [comment]
LGTM

**Comment by vieux** [comment]
thanks @jfrazelle

**Comment by jfrazelle** [comment]
in favor of getting the docs updated so people can try it out

**Pull request merged**